endpoint/endpoint_core.c

```diff
@@ -23,9 +23,18 @@
 #define STR_HELPER(num) #num
 #define STR(num) STR_HELPER(num)
 
-static const char* HOST = STR(ENDPOINT_HOST);
-static const char* PORT = STR(ENDPOINT_PORT);
-static const char* SSL_SEED = STR(ENDPOINT_SSL_SEED);
+#ifndef EP_TA_HOST
+#define EP_TA_HOST localhost
+#endif
+#ifndef EP_TA_PORT
+#define EP_TA_PORT 8000
+#endif
+#ifndef EP_SSL_SEED
+#define EP_SSL_SEED nonce
+#endif
+static const char* HOST = STR(EP_TA_HOST);
```

Comment:
These variables can be eliminated since they are used once.

Reply:
Done.